src/wp-content/themes/twentythirteen/functions.php

```diff
@@ -319,7 +319,7 @@ function twentythirteen_scripts_styles() {
 	wp_enqueue_style( 'genericons', get_template_directory_uri() . '/genericons/genericons.css', array(), '3.0.3' );
 
 	// Loads our main stylesheet.
```

Comment:
It looks like several other styles and scripts did not have their versions bumped. I had always just assumed that the theme version was used, but seems that for older default themes, that's not the case.

@peterwilsoncc what do you think about changing all of these date versions to be accurate now? Or, we could just open a follow up ticket.